Marlin/ultralcd.cpp

```diff
@@ -3460,17 +3457,15 @@ void lcd_update() {
   #if ENABLED(ULTIPANEL)
     static millis_t return_to_status_ms = 0;
     manage_manual_move();
-  #endif
 
-  lcd_buttons_update();
+    lcd_buttons_update();
```

Comment:
But main dev is like mychange. So I was guessing it was a fault of you. If no fault, I believe that a if def is needed here.